src/pages/KadraVykhovnykiv/KVTable.tsx

```diff
@@ -70,6 +71,8 @@ export const KVTable = ({ current, searchData }: props) => {
   }
 
   const fetchData = async () => {
+    if (searchPage != 0)
+    setPage(searchPage);
    const dir: string = (sortKey > 0)? 'ascend' : 'descend';
    var col = 'id';
    switch (sortKey) {
```

Comment:
I think it is not your code, but I believe it would make more sense to 
`switch(Math.abs(sortKey))` 
to remove those:
`case number:
case -number: `

Reply:
Also, not sure why there is such uncertainty with sortKey. (why it can be either positive or negative for the same value)
Probably, this part should be revised

Reply:
positive sortKey means ascend, negative - descend.
"switch(Math.abs(sortKey)) " - fixed

Reply:
Thanks, missed this one